heart-attack.html

```diff
@@ -0,0 +1,299 @@
+				Al predecir si alguien tendrá un segundo ataque al corazón, no tendremos el valor de la variable objetivo, por lo que no
+				podemos realizar un análisis de la precisión de los resultados. El dataset se dividirá en 70-30% de forma aleatoria.
+			  </p>
+			  <pre>
+				<code>
+X = df.loc[:,df.columns != '2do_Ataque_Corazon']
+y = df['2do_Ataque_Corazon'].values
+train_X, test_X, train_y, test_y = train_test_split(X, y, test_size=0.30,random_state=0, shuffle=True)
+				</code>
+			  </pre>
+
+			  <p>
+				Ahora debemos entrenar y aplicar el modelo.
+			  </p>
+
+			  <pre>
+				<code>
+lr = LogisticRegression()
+lr.fit(train_X, train_y)
+
+y_pred = lr.predict(test_X)
+				</code>
+			  </pre>
+
+			  <p>
+				Imprimir la Matriz de Confusión y el Reporte de Clasificación nos dará la información sobre la precisión de nuestro modelo.
+			  </p>
+
+			  <pre>
+				<code>
+print(classification_report(test_y, y_pred, digits=3))
+print(confusion_matrix(test_y, y_pred))
+				</code>
+	  		  </pre>
+			
+				<pre>
+					<code>
+               precision    recall  f1-score   support
+
+          No      0.941     0.842     0.889        19
+          Si      0.880     0.957     0.917        23
+
+    accuracy                          0.905        42
+   macro avg      0.911     0.899     0.903        42
+weighted avg      0.908     0.905     0.904        42
+
+[[16  3]
+[ 1 22]]
+					</code>
+				</pre>
+			
+				<p>
+					Vemos que en total obtuvimos únicamente 4 predicciones erróneas. Este número puede variar dependiendo de las muestras
+					aleatorias en cada conjunto, pero es una baja cantidad de fallos. Esto indica que el modelo se comporta correctamente.
+				</p>
+
+			  <h3 class="major">Aplicación del modelo</h3>
+
+			  <p>
+				Sabiendo que el modelo funciona, podemos ahora aplicarlo a el dataset de testing. A continuación se provee el código
+				modificado para aplicar el modelo sobre el dataset deseado.
+			  </p>
+
+			  <pre>
+				<code>
+import pandas as pd
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import confusion_matrix, classification_report
+from sklearn.model_selection import train_test_split
+
+input_file = "cardiac-training.csv"
+df = pd.read_csv(input_file, header=0)
+input_file_scoring = "cardiac-scoring.csv"
+df_scoring = pd.read_csv(input_file_scoring, header=0)
+
+X = df.loc[:, df.columns != '2do_Ataque_Corazon']
+y = df['2do_Ataque_Corazon'].values
+
+lr = LogisticRegression()
+lr = lr.fit(X, y)
+
+X_scoring = df_scoring.loc[:]
+y_pred = lr.predict(X_scoring)
+				</code>
+  			  </pre>
+
+				<p>
+					Nuestras predicciones fueron realizadas. Ahora es nuestro trabajo, dados los inputs, analizar si estas tienen sentido. Por ejemplo,
+					el ejemplo 11. La predicción es que 'Si'. El paciente tiene alto sobrepeso, es de la tercera edad con alto colesterol y alto nivel
```

Comment:
Nit: pondría un 'de' acá
```suggestion
					el ejemplo 11. La predicción es que 'Si'. El paciente tiene alto sobrepeso, es de la tercera edad con alto colesterol y alto nivel de
```